lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -1078,6 +1082,7 @@ def get_proposed(self):
         of proposed commands.
         Return a list of raw cli command strings.
         """
+        cmds = []
```

Comment:
note that since this isn't used until the return from this function, the standard python idiom would be to not define it up here but to let it be created at the very bottom.

Reply:
That's true.  This should have been removed.  Would you like me to make this change before this goes into stable-2.9?

Reply:
Updating now

Reply:
Sure.  Since this PR also has to be updated to resolve the conflicts and may need the ```**kwargs``` change as well, this would be good to get in.